plugins/hydra_ax_sweeper/tests/config/params/nested_with_escape_char.yaml

```diff
@@ -0,0 +1,11 @@
+hydra:
+    sweeper:
+      params:
+        ax_config:
+          params:
+            quadratic_a\_rgs_x:
```

Comment:
I think you need two app config options:
```yaml
quadratic:
  x: ???
  y: ???
```
and:
```yaml
quad_x: ???
quad_y: ???
```

for the first one, the corresponding params looks like (in compact fucking notation, yeah!):
```yaml
hydra.sweeper.params.ax_config.params:
  quadratic_x:
    type: range
    bounds: [-1, 1]
  quadratic_y:
    type: range
    bounds: [-1, 1]
```
Command line overrides for this one should look like:
```
$ python foo.py quadratic.x=-5.0:5.0 quadratic.y=-5.0:5.0
```

for the second one, the corresponding params looks like:
```yaml
hydra.sweeper.params.ax_config.params:
  quadratic_x\_arg:
    type: range
    bounds: [-1, 1]
  quadratic_x\_arg:
    type: range
    bounds: [-1, 1]
```
Command line overrides for this one should look like:
```
$ python foo.py quadratic.x_arg=-5.0:5.0 quadratic.y_arg=-5.0:5.0
```

Reply:
updated above (had an error in the first app config example)

Reply:
Please change to my configs, it will be much easier to read.